schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -996,6 +1000,10 @@ public void close() {
         client.close();
     }
 
+    public SchemaRegistryVersion clientVersion() {
+        return CLIENT_VERSION;
+    }
+
     private <T> T createInstance(SerDesInfo serDesInfo, boolean isSerializer) {
```

Comment:
Should we expose a function to get the SchemaRegistryVersion of the server to which the client will be interacting with ?

Reply:
@raju-saravanan I thought about that while adding `clientVersion` API to add a new API like `serverVersion` with default method in `ISchemaRegistryService` but left that as I did not see any need for now.

Reply:
ok